services/snapcraft/snapcraft-base.js

```diff
@@ -0,0 +1,23 @@
+import { BaseJsonService, pathParam } from '../index.js'
+
+export const snapcraftPackageParam = pathParam({
+  name: 'package',
+  example: 'Redis',
```

Comment:
One more quick thing I noticed. Package names are case sensitive Redis with a captial R is not found

```suggestion
  example: 'redis',
```